tests/data/token_indexers/pretrained_transformer_indexer_test.py

```diff
@@ -99,7 +99,7 @@ def test_transformers_vocab_sizes(self, model_name):
 
     def test_transformers_vocabs_added_correctly(self):
         namespace, model_name = "tags", "roberta-base"
-        tokenizer = cached_transformers.get_tokenizer(model_name)
+        tokenizer = cached_transformers.get_tokenizer(model_name, use_fast=False)
```

Comment:
Why `use_fast=False`?

Reply:
`RobertaTokenizerFast` does not have the attribute `encoder` which we use in this test case.